docs/piscine-rust-curriculum.md

```diff
@@ -10,14 +10,14 @@ Exercices:
 
```

Comment:
As this is your branch can you please Change 
"Exercices:" to "Exercises:"

Reply:
kk

Reply:
@MSilva95 done